README.md

```diff
@@ -1,15 +1,76 @@
-# Fast Store
+# FastStore 
+## An open ecommerce framework
 
-https://docs.google.com/presentation/d/10SCY7ZO1WWGeUQs6VzJWAlp4r1404dVtIxx_YgYldZI/edit#slide=id.g8ef796eccd_0_336
+FastStore is an open ecommerce framework built using modern technologies that aims to deliver the building blocks necessary for creating your custom ecommerce experience. With these building blocks you can archieve:
```

Comment:
"Ship really fast stores using technologies you already know, CI/CD integration, and observability about your development work.

Store Framework Jamstack is an iteration over the VTEX open ecommerce framework. Built using modern technologies, it aims to deliver the assets necessary for a great storefront user experience. With these assets, you can achieve:"